exporters/otlp/src/otlp_populate_attribute_utils.cc

```diff
@@ -315,6 +316,22 @@ void OtlpPopulateAttributeUtils::PopulateAttribute(
   }
 }
 
+void OtlpPopulateAttributeUtils::PopulateAttribute(
+    opentelemetry::proto::common::v1::InstrumentationScope *proto,
+    const opentelemetry::sdk::instrumentationscope::InstrumentationScope
+        &instrumentation_scope) noexcept
+{
+  if (nullptr == proto)
+  {
+    return;
+  }
```

Comment:
Please change to `assert(proto != nullptr)` instead if you feel it must stay in the code, or just remove.

If there is a code path invoking PopolateAttribute() with a nullptr, the caller should be fixed instead, although I have not seen any.

Reply:
Thanks. I've removed the check.